components/http_foundation.rst

```diff
@@ -377,6 +377,14 @@ method takes an instance of
 You can clear a cookie via the
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::clearCookie` method.
 
+You can also create a 
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` object from a raw header value using :method:`Symfony\Component\HttpFoundation\Cookie::fromString`.
+
+.. tip::
```

Comment:
I'm not sure about this TIP. Don't you think it's too low level? And it's not really a Symfony feature, but a PHP / HTTP feature, right?